pkg/workceptor/command.go

```diff
@@ -239,6 +240,35 @@ func (cw *commandUnit) Start() error {
 	return cw.runCommand(cmd)
 }
 
+// isOrphaned checks if command runner for this unit exists
+func (cw *commandUnit) isOrphaned() (bool, error) {
+	pid := cw.Status().ExtraData.(*commandExtraData).Pid
+	p, err := process.NewProcess(int32(pid))
+	if err != nil {
+		// process does not exist
+		return true, err
```

Comment:
Im finding the name of the function a little confusing here. If the process dosent exist, how can it also be an orphaned process?

Perhaps changing the name to something like "isPidValid()" and reversing the bools returned will make a little more sense? What do you think?